tests/bson-deserialize-typewrapped-utcdatetime-001.phpt

```diff
@@ -0,0 +1,101 @@
+--TEST--
+BSON serialization: type wrapped UTCDateTime
+--FILE--
+<?php
+$tests = [
+	[ 'types' => [] ],
+	[ 'types' => [ 'UTCDateTime' => 'MyUTCDateTime' ] ],
+	[ 'root' => 'array', 'types' => [ 'UTCDateTime' => 'MyUTCDateTime' ] ],
+	[ 'types' => [ 'UTCDateTime' => 'MyUTCDateTimeReturnDateTimeType' ] ],
+];
+
+class MyUTCDateTime implements \MongoDB\BSON\TypeWrapper
```

Comment:
Implement UTCDateTimeInterface. I suppose you might also want tests for cases where the classes don't implement the type-specific interface.